### PR TITLE
update README.md with a capital D for domotica and a period

### PR DESCRIPTION
**Pull request by VJagram**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,2 @@
 # domotica
-Project domotica voor groep 5 haagse hogeschool Delft
+Project Domotica voor groep 5 haagse hogeschool Delft.
```
Comment (BenjaminKnol) [inline README.md]: shouldnt this be Haagse Hogeschool Delft?

**Review by BenjaminKnol — COMMENTED** [review]
Before merge please fix comment

File: README.md (hunk 1 above)
Comment (VJagram) [reply README.md]: Yes it is. I will fix and update the pull request.

Comment (VJagram) [reply README.md]: I've fixed the capitalisation issue.

**Review by BenjaminKnol — APPROVED** [review]
Quick fix shame no one noticed early, good job!

**Pull request merged**